api/reviewer_api/resources/foiflowmasterdata.py

```diff
@@ -257,13 +257,14 @@ def post(ministryrequestid):
 
 @cors_preflight("POST,OPTIONS")
 @API.route("/foiflow/oss/presigned/redline/<int:ministryrequestid>")
+@API.route("/foiflow/oss/presigned/redline/<int:ministryrequestid>/<string:redlinetype>")
```

Comment:
I used the same route as redline, but this will save the oipc package to an 'oipc' folder instead

Reply:
Sounds good. Made minor changes to endpoint to make the redactionlayer as path param,
@API.route("/foiflow/oss/presigned/<redactionlayer>/<int:ministryrequestid>/<string:layertype>")